### ISSUE TITLE
[zig fmt] failure to format async function pointer

### ISSUE THREAD

**Issue by emekoi** [ziglang/zig#2521]
```
std\event\io.zig:15:77: error: Expected parameter list, found Comma
        readFn: async<*Allocator> fn (self: *Self, buffer: []u8) Error!usize,
                                                                            ~
std\event\io.zig:15:17: error: Expected type expression, found Keyword_async
        readFn: async<*Allocator> fn (self: *Self, buffer: []u8) Error!usize,
                ~~~~~
std\event\net.zig:11:94: error: Expected parameter list, found Comma
    handleRequestFn: async <*mem.Allocator> fn(*Server, *const std.net.Address, os.File) void,
                                                                                             ~
std\event\net.zig:11:22: error: Expected type expression, found Keyword_async
    handleRequestFn: async <*mem.Allocator> fn(*Server, *const std.net.Address, os.File) void,
                     ~~~~~
```
this will be solved by #2377 correct?

**Comment by andrewrk** [comment]
> this will be solved by #2377 correct?

Yes, that syntax is going to change.

**Comment by Vexu** [comment]
This has been fixed.
